src/main/java/com/jaspersoft/jasperserver/jaxrs/client/apiadapters/authority/organizations/OrganizationsService.java

```diff
@@ -45,4 +45,11 @@ public SingleOrganizationAdapter organization(String organizationId) {
     public BatchOrganizationsAdapter allOrganizations() {
         return new BatchOrganizationsAdapter(sessionStorage);
     }
+
+    // Deprecated method, use allOrganizations() method
+@Deprecated
```

Comment:
Please use @deprecated in javadoc (with suggestion) like here: http://www.atetric.com/atetric/javadoc/net.sf.jasperreports/jasperreports/5.6.0/src-html/net/sf/jasperreports/engine/JRExporterParameter.html